developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -387,6 +387,155 @@
+    </Control>
+
+    <Control Name="cmdResetToolWindows" Title="General tab">
+      <p></p>
+    </Control>
+
+    <Control Name="cmdSMTPSettings" Title="SMTP Settings...">
+      <p>Configure your SMTP server settings for email functionality within Keyman Developer, 
+      such as emailing debug URLs for the touch layout debugger.</p>
+    </Control>
+
+    <Control Name="cmdProxySettings" Title="Proxy Settings...">
+      <p>Configure the HTTP proxy settings for online functionality within Keyman Developer, 
+      such as uploading files to Tavultesoft.</p>
+    </Control>
+
+    <Control Name="cmdCancel" Title="Options Dialog">
+      <p>Click Cancel to close the tab.</p>
+    </Control>
+
+    <Control Name="cmdOK" Title="Options Dialog">
+      <p>Click OK to save changes.</p>
+    </Control>
+
+    <Control Name="chkUseTab" Title="Use tab character">
+      <p>Sets whether to use the Tab character or spaces for indents.</p>
+    </Control>
+
+    <Control Name="editIndentSize" Title="Indent size">
+      <p>The indent size, measured in number of spaces.</p>
+    </Control>
+
+    <Control Name="chkLinkFontSizes" Title="Link quoted font size to primary font size">
+      <p>If checked, the quoted font size will be the same as the default font size.</p>
+    </Control>
+
+    <Control Name="cmdDefaultFont" Title="Default font">
+      <p>Sets the font for ordinary text in the editor.</p>
+    </Control>
+
+    <Control Name="cmdQuotedFont" Title="Quoted font">
+      <p>Sets the font for displaying comments text and strings in Keyman source files in single or double quotes.</p>
+    </Control>
+
+    <Control Name="cbEditorTheme" Title="Editor theme">
+      <p>Sets the display theme and syntax highlighting options for the editor. A custom theme can be defined in a JSON file.</p>
+    </Control>
+
+    <Control Name="chkUseOldDebugger" Title="Enable test window (from version 5.0)">
+      <p>Enables the Keyman 5-style Test window instead of the debugger.</p>
+    </Control>
+
+    <Control Name="chkDebuggerBreakWhenExitingLine" Title="Breakpoints fire also when exiting line">
+      <p>Breakpoints will fire when a rule has finished being processed also.</p>
+    </Control>
+
+    <Control Name="chkDebuggerSingleStepAfterBreak" Title="Turn on single step after breakpoint">
+      <p>Activates single-step mode after a breakpoint fires.</p>
+    </Control>
+
+    <Control Name="chkDebuggerShowStoreOffset" Title="Show matched character offsets in stores">
+      <p>Displays numeric indices of characters in stores</p>
+    </Control>
+
+    <Control Name="chkDebuggerAutoRecompile" Title="Automatically recompile if no debug information available">
+      <p>When the debugger starts, rebuild a keyboard without prompting if the keyboard 
+      has no debug symbols included.</p>
+    </Control>
+
+    <Control Name="chkDebuggerAutoResetBeforeCompiling" Title="Automatically reset debugger before recompiling">
+      <p>update is coming</p>
```

Comment:
This seems unfinished?

Reply:
I copied it from the [here](https://help.keyman.com/developer/17.0/context/options#:~:text=Automatically%20reset%20debugger%20before%20recompiling).

Reply:
What can I put here instead?

Reply:
How about: 

> If you are debugging a keyboard, make a change to the keyboard, and recompile it, then the debugger needs to be reset in order to get access to the new keyboard. Keyman Developer will normally prompt you to do this, but if you set this option, then it will reset the debugger automatically instead.

Reply:
Noted!

Reply:
I'll also update it in help.keyman.com.